### PR TITLE
G01_2_4: Add backend connection for emotion assessment (frontend)

### PR DESCRIPTION
**Pull request by shirokov-kirill**

### CODE REVIEW

File: app-web-frontend/src/setupInfo.js
Comment (antoniibelyshev) [inline app-web-frontend/src/setupInfo.js:1]: from launch.json:
> "url": "http://localhost:3000"
is that ok that here the server address is different?

File: app-web-frontend/src/index.js
Comment (antoniibelyshev) [inline app-web-frontend/src/index.js:1]: It's better to leave this file as is in development and add you functionality to Header.js and App.js

File: app-web-frontend/src/setupInfo.js
Comment (shirokov-kirill) [reply app-web-frontend/src/setupInfo.js:1]: launch.json is a file to debug frontend in vscode. This url corresponds to react-app (frontend)

File: app-web-frontend/src/index.js
Comment (shirokov-kirill) [reply app-web-frontend/src/index.js:1]: Agree

**Pull request merged**